src/main/resources/application.yaml

```diff
@@ -28,6 +28,24 @@ no.nav.security.jwt:
     discoveryurl: ${AZURE_APP_WELL_KNOWN_URL}
     accepted_audience: ${AZURE_APP_CLIENT_ID}
   client.registration:
+    aad-graph-onbehalfof:
+      resource-url: ${AAD_GRAPH_API_URI}
+      token-endpoint-url: ${AZURE_OPENID_CONFIG_TOKEN_ENDPOINT}
+      grant-type: urn:ietf:params:oauth:grant-type:jwt-bearer
+      scope: ${AAD_GRAPH_SCOPE}
```

Comment:
Vi trenger å definiere `AAD_GRAPH_API_URI` og `AAD_GRAPH_SCOPE` også
`AAD_GRAPH_API_URI: https://graph.microsoft.com/v1.0/`
`AAD_GRAPH_SCOPE=https://graph.microsoft.com/.default`

Skal ikke være behov for å legge inn den i outbound https://doc.nais.io/workloads/reference/access-policies/?h=graph.microsoft.com

Tror vi også må be om tilgang til tjenesten, i #tech-azure